onboarding_guide_outline.md

```diff
@@ -67,15 +67,31 @@ Link to snapshot of the server running
 
 Before we try to make this faster, let's measure it.
 
-Add a local_resource that records the current time, and kicks off a build.
+Tilt can run commands locally.
```

Comment:
This whole section of the template seems scattered. e.g. this line isn't helpful without context. i'd strike it or replace with: "in addition to [running things]|[doing magic] in your cluster, Tilt also gives you tools to run commands locally."

Consider:
> In addition to [running things]|[doing magic] in your cluster, Tilt also gives you tools to run commands locally.

> We want to measure the time from code change to new process. To do that, we'll be making use of [local_resource](link-to-doc), which lets you locally run scripts, shell code, or servers, and manage them from your sidebar like any other Tilt resource.

> First, we add...

Reply:
oh, ya, totes agree, this is intended to be an outline that hits the key story beats; I expect an outline to be pretty scattered! Added some parens on this

Reply:
sure, I just think that this bit is consistent enough across guides that we can provide the actual copy